src/cli.js

```diff
@@ -73,6 +75,11 @@ Options:
     });
   }
 
+  if (options.maintenance) {
```

Comment:
@RamonGebben this should not be `options.maintenance`, but `options.environment && options.destroy`, isn't?

Reply:
🤭  You're totally right. I've corrected it 😅